analysis/deep2_r23_o32/plotting/te_metal_plots.py

```diff
@@ -311,23 +319,15 @@ def jiang_comparison(fitspath):
 
 
     jR23_det = jiang_calibration(metal_det, lO32)
-    A_comparison = np.zeros(len(metal_det))
-    count = len(metal_det) + len(der_R23) + len(der_R23_MACT)
+    A_comparison = (jR23_det - lR23)
+    count = len(metal_det) + len(der_r23) + len(der_r23_mact)
     print(count)
 
-    for ii in range(len(metal_det)):
-        A_comparison[ii]= (jR23_det[ii] - lR23[ii])
+    jR23_DEEP = jiang_calibration(der_oh, der_o32)
+    jR23_MACT = jiang_calibration(der_oh_mact, der_o32_mact)
 
-    jR23_DEEP = jiang_calibration(der_OH, der_O32)
-    jR23_MACT = jiang_calibration(der_OH_MACT, der_O32_MACT)
-
-    B_comparison = np.zeros(len(der_R23))
-    C_comparison = np.zeros(len(der_R23_MACT))
-    for bb in range(len(jR23_DEEP)):
-        B_comparison[bb]= (jR23_DEEP[bb] - der_R23[bb])
-
-    for aa in range(len(jR23_MACT)):
-        C_comparison[aa]= (jR23_MACT[aa] - der_R23_MACT[aa])
+    B_comparison = jR23_DEEP - der_r23
```

Comment:
Perhaps use consistent variable names so that `der_r23` is clearly known to be DEEP2.

Reply:
changed to deep2_r23